diff -up audacity-sources-3.4.1/libraries/lib-time-and-pitch/StaffPad/SimdTypes_neon.h.1~ audacity-sources-3.4.1/libraries/lib-time-and-pitch/StaffPad/SimdTypes_neon.h
--- audacity-sources-3.4.1/libraries/lib-time-and-pitch/StaffPad/SimdTypes_neon.h.1~	2023-11-08 23:13:52.496674319 +0100
+++ audacity-sources-3.4.1/libraries/lib-time-and-pitch/StaffPad/SimdTypes_neon.h	2023-11-08 23:14:57.753140302 +0100
@@ -45,7 +45,7 @@ struct float_x4
     s.n128_f32[2] = v2;
     s.n128_f32[3] = v3;
 #elif __clang__
-    s = {v0, v1, v2, v3};
+    s = float32x4_t{v0, v1, v2, v3};
 #else
     float f[4] = {v0, v1, v2, v3};
     s = vld1q_f32(f);
