diff -up audacity-4.0.0/muse/buildscripts/cmake/SetupQt6.cmake.omv~ audacity-4.0.0/muse/buildscripts/cmake/SetupQt6.cmake
--- audacity-4.0.0/muse/buildscripts/cmake/SetupQt6.cmake.omv~	2026-09-03 00:00:00.000000000 +0000
+++ audacity-4.0.0/muse/buildscripts/cmake/SetupQt6.cmake	2026-09-03 00:00:00.000000000 +0000
@@ -91,7 +91,7 @@ include(QtInstallPaths)
 
 message(STATUS "Qt version: ${Qt6_VERSION}")
 
-qt_standard_project_setup(REQUIRES 6.8 SUPPORTS_UP_TO 6.10)
+qt_standard_project_setup(REQUIRES 6.8 SUPPORTS_UP_TO 6.11)
 
 if (QT_IS_STATIC)
     qt_add_library(all_qml_plugins STATIC)
