diff -uraN audacity-sources-3.3.1/CMakeLists.txt omv-audacity-sources-3.3.1/CMakeLists.txt
--- audacity-sources-3.3.1/CMakeLists.txt	2023-04-27 19:18:56.000000000 +0200
+++ omv-audacity-sources-3.3.1/CMakeLists.txt	2023-04-30 15:20:13.097171674 +0200
@@ -420,7 +420,7 @@
 # Setup RPATH handling
 set( CMAKE_BUILD_RPATH "$ORIGIN/../${_PKGLIB}" )
 set( CMAKE_BUILD_WITH_INSTALL_RPATH FALSE )
-set( CMAKE_INSTALL_RPATH "$ORIGIN/../${_PKGLIB}" )
+set( CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${_PKGLIB}" )
 set( CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE )
 
 # Adjust them for the Mac
