diff -up audacity-sources-3.2.1/src/CMakeLists.txt.1~ audacity-sources-3.2.1/src/CMakeLists.txt
--- audacity-sources-3.2.1/src/CMakeLists.txt.1~	2022-10-26 17:59:12.928352010 +0200
+++ audacity-sources-3.2.1/src/CMakeLists.txt	2022-10-26 18:04:25.865045721 +0200
@@ -1483,16 +1483,17 @@ else()
    configure_file( audacity.desktop.in ${_INTDIR}/audacity.desktop )
 
    # Copy the required wxWidgets libs into the bundle
-   add_custom_command(
-      TARGET
-         ${TARGET}
-      COMMAND
-         ${CMAKE_COMMAND} -D SRC="${_EXEDIR}/audacity"
-                          -D DST="${_DEST}/${_PKGLIB}"
-                          -D WXWIN="${_SHARED_PROXY_BASE_PATH}/$<CONFIG>"
-                          -P ${AUDACITY_MODULE_PATH}/CopyLibs.cmake
-      POST_BUILD
-   )
+   # *** Nobody in their right mind would want to do this ***
+   #add_custom_command(
+   #   TARGET
+   #      ${TARGET}
+   #   COMMAND
+   #      ${CMAKE_COMMAND} -D SRC="${_EXEDIR}/audacity"
+   #                       -D DST="${_DEST}/${_PKGLIB}"
+   #                       -D WXWIN="${_SHARED_PROXY_BASE_PATH}/$<CONFIG>"
+   #                       -P ${AUDACITY_MODULE_PATH}/CopyLibs.cmake
+   #   POST_BUILD
+   #)
 
    add_executable(findlib ../linux/findlib.c)
    target_link_libraries(findlib ${CMAKE_DL_LIBS})
